INSTALL.md

```diff
@@ -6,7 +6,7 @@ In a nutshell, to install IELE on an Ubuntu 16.04 machine, run:
 ```
 
 sudo apt-get update
-sudo apt-get install make gcc maven curl openjdk-8-jdk flex opam pkg-config libmpfr-dev autoconf libtool pandoc libssl-dev build-essential libffi-dev libpython-dev python-setuptools
+sudo apt-get install make gcc maven curl openjdk-8-jdk flex opam pkg-config libmpfr-dev autoconf libtool pandoc libssl-dev build-essential libffi-dev libpython-dev python-setuptools libssl-dev
 git submodule update --init # Initialize submodules
```

Comment:
I undetstand this was already approved/merged, but it seems you already had libssl-dev earlier in the list: does the order matter? Should the earlier occurrence be removed? It looks a bit sloppy if we repeat this unnecessarily.